packages/babel-parser/src/plugins/jsx/index.js

```diff
@@ -327,8 +327,21 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         node.expression = this.jsxParseEmptyExpression();
       } else {
         node.expression = this.parseExpression();
```

Comment:
Is there a reason to check if the expression is parenthesized instead of using `parseMaybeAssign` (which already disallow comma expressions) here?

Reply:
👍